src/handlers/member.ts

```diff
@@ -53,6 +53,16 @@ export async function addRole(
   roleID: string,
   reason?: string,
 ) {
+  const guild = await cacheHandlers.get("guilds", guildID);
+  if (!guild) return;
+  
+  if (guild.ownerID === botID) {
```

Comment:
Could we replace this with 
```ts
if (guild && guild.ownerID === botID)
```

Reply:
Instead of if (!guild) this would allow us to continue checking and using these without needing guild in cache stored